src/Views/Page.elm

```diff
@@ -88,6 +91,26 @@ stagingAlert { session, loadUrl } =
         text ""
 
 
+newVersionAlert : Config msg -> Html msg
+newVersionAlert { session, reloadPage } =
+    case session.currentVersion of
+        Request.Version.NewerVersion currentVersion newVersion ->
+            div [ class "NewVersionAlert d-block align-items-center" ]
+                [ text "Il y a une nouvelle version de l'application disponible. Vous pouvez la voir en "
+                , a
+                    [ href "./"
+                    , onClick reloadPage
+                    ]
+                    [ text "rafraîchissant cette page" ]
+                , text ". Vous pouvez aussi consulter le "
+                , a [ href <| "https://github.com/MTES-MCT/wikicarbone/compare/" ++ currentVersion ++ ".." ++ newVersion ]
```

Comment:
Not sure it's useful enough atm to link to changelog in its current state. We'll have to make a non-technical, French version of it when we'll have tags/releases anyway.